File: Chapter2/FuzzyProteins/ScriptForGR/Python/1-mkdir.py
import os

lista = open("/home/maria/Documentos/InterfaceRes/ListaRedundant_Unique")
os.system("cd /home/maria/Documentos/InterfaceRes/; mkdir VPs")

linea = lista.readline()
while linea:
    linea = linea.strip()
    splista = linea.split(" ")
    # splista[0] = splista[0].lower()
    os.system("cd /home/maria/Documentos/InterfaceRes/VPs; mkdir " + splista[0])
    linea = lista.readline()

lista.close()

File: Chapter2/FuzzyProteins/ScriptForGR/Perl/1-mkdir.pl
use strict;

open(lista,"/home/maria/Documentos/InterfaceRes/ListaRedundant_Unique");
system("cd /home/maria/Documentos/InterfaceRes/; mkdir VPs");

my $lista=<lista>;
while($lista=<lista>){
	chomp $lista;
	my @splista=split " ",$lista;
	#@spb[0]=lc(@spb[0]);
	system("cd /home/maria/Documentos/InterfaceRes/VPs;mkdir @splista[0]");
}

close(lista);
